### ISSUE TITLE
local variable 'changed' referenced before assignment

### ISSUE THREAD

**Issue by villain** [MITRECND/WhoDat#9]
i have successfully done an import using elasticsearch (started with v6), however when trying to import another version (doesnt matter which), the import fails with the following message:

Exception in thread Worker 0:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "./elasticsearch_populate.py", line 167, in process_worker
    process_entry(insert_queue, es, work['header'], work['row'], options)
  File "./elasticsearch_populate.py", line 246, in process_entry
    for ch in changed:
UnboundLocalError: local variable 'changed' referenced before assignment

ive tried importing several different versions (v7, v8, v9, v4, v2..) and get the same error each time

**Comment by wxsBSD** [comment]
Can you please provide the command you used? Looks like you may be using the "exclude" option when importing? If that is true it looks like we are missing the changed and diff sets from that branch.

**Comment by villain** [comment]
confirming i was running it with the exclude option

**Comment by wxsBSD** [comment]
This should be fixed once the elastic search branch is merged in.

**Comment by Mraoul** [comment]
The branch was merged into master earlier today, give us a heads up if its
still broken.
